tests/integration/WatchdogTest.php

```diff
@@ -37,6 +37,39 @@ public function testWatchdog()
         $output = $this->getOutput();
         $this->assertGreaterThanOrEqual($message_chars, substr_count($output, $char));
 
+        // Test deleting a watchdog message by filtering on text.
```

Comment:
Keeping this commented out seems fine to me, for now. Would it make sense to move down to "// Tests message deletion" section?

Reply:
ha ha, I was deleting it there, early, for convenience when developing the `wd-show` assertions, to reduce the screen output by removing that long message.  I wanted to keep the original notice message.

Reply:
Decided to expand the delete testing and split the function into two.

Reply:
The debug output could be removed if we don't want that. But I thought it might illuminate something about why the tests fails on some builds.